sdk/tests/tutorials/properties/test_transaction_properties.py

```diff
@@ -0,0 +1,145 @@
+# import general python packages
+import unittest
+import json
+from datetime import datetime
+import pytz
+import logging
+
+# import lusid specific packages
+import lusid
+import lusid.models as models
+from utilities import InstrumentLoader
+from utilities import TestDataUtilities
+from lusidtools.logger import LusidLogger as logger
```

Comment:
This will cause a circular dependency as lpt depends on the sdk. You just need to initialise the logger in your setup code using the standard logging

Reply:
removed lpt logger